FluentTc.Tests/ProjectRetrievalTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using FakeItEasy;
+using FluentTc.Domain;
+using FluentTc.Engine;
+using NUnit.Framework;
+
+namespace FluentTc.Tests
+{
+    [TestFixture]
+    public class ProjectRetrievalTests
```

Comment:
I will move these to AcceptanceTests and will make better overall tests for the method. I started here when I was trying t make more robust tests, and did not move these out.